go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	github.com/vmware/govmomi v0.22.2
```

Comment:
Was there any go sum changes with this?

Reply:
No. The only thing that has changed (vendoring-wise) is that hive is now directly importing packages from github.com/vmware/govmmomi instead of calling into github.com/openshift/installer to test connectivity to a vsphere endpoint.